src/LineageTree/lineageTree.py

```diff
@@ -92,7 +93,7 @@ def add_branch(
         """
         if length == 0:
             return pred
-        if self.predecessor.get(pred, None) and not reverse:
+        if self.predecessor.get(pred) and not reverse:
```

Comment:
```python
if pred in self.predecessor and not reverse:
    ...
```
if there is no empty `list`

Reply:
does not work for some reason

Reply:
There is no error it just runs the warning without reason specifically when I modify branch